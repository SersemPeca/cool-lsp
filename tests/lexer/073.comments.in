"first"
-- single line comments are also ignored
"second"
