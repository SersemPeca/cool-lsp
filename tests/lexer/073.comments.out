#1 STR_CONST "first"
#3 STR_CONST "second"
